setup.py

```diff
@@ -47,16 +47,10 @@
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Framework :: Django',
-        'Framework :: Django :: 1.11',
-        'Framework :: Django :: 2.0',
-        'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        "Programming Language :: Python :: 3",
```

Comment:
We want to keep this one, some tools in the Python ecosystem use it to determine whether the package supports any version of Python 3.